src/auto-trace.js

```diff
@@ -10,6 +10,10 @@ export function addGlobalMiddleware(middlewareFn) {
 	globalMiddlewares.push(middlewareFn)
 }
 
+export function removeAllGlobalMiddlewares(){
```

Comment:
This is mainly so that tests can reset the middleware; figured others may want to reset the middleware as well.